api-client/src/utils/textTruncator.ts

```diff
@@ -0,0 +1,10 @@
+const textTruncator = (text: string, numOfCharsAllowed: number, replaceLineBreaks = true): string => {
+  const ellipsis = '...'
+  const cleanText = replaceLineBreaks ? text.replace(/\n/g, ' ') : text
+  if (cleanText.length < numOfCharsAllowed) {
```

Comment:
I think this should be
```suggestion
  if (cleanText.length.trim() <= numOfCharsAllowed) {
```